app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue

```diff
@@ -227,6 +227,28 @@ export default {
     },
   },
 
+  watch: {
+    '$route.name'() {
+      const routeName = this.$route?.name;
+      const routeParams = this.$route?.params;
+      if (routeName === 'list_all_portals') {
+        this.fetchPortalAndItsCategories();
+      }
+      if (
```

Comment:
@sojan-official @pranavrajs Any better way to show the article count in real-time by updating the socket? Here I added a param in the route when creating and deleting an article, and it will fetch portal details if it is present.

Reply:
@sojan-official Is there a socket available for the counts by locales?

Reply:
We don't have WebSockets available for the articles model.